test/upgrade/prober/configuration.go

```diff
@@ -62,22 +64,49 @@ func (p *prober) annotateNamespace() {
 	ensure.NoError(err)
 }
 
+func (p *prober) fetchBrokerUrl() (*apis.URL, error) {
+	namespace := p.config.Namespace
+	p.log.Debugf("Fetching %s broker URL for ns %s", brokerName, namespace)
+	meta := resources.NewMetaResource(brokerName, p.config.Namespace, lib.BrokerTypeMeta)
+	err := duck.WaitForResourceReady(p.client.Dynamic, meta)
+	if err != nil {
+		return nil, err
+	}
+	brokers := p.client.Eventing.EventingV1alpha1().Brokers(namespace)
```

Comment:
if we already know the specific `brokerName` we want, could we please combine this into a single line, ie:
`broker, err := p.client.Eventing.EventingV1alpha1().Brokers(namespace).Get(brokerName, metav1.GetOptions{})
if err != nil {
  return nil, err
}

Reply:
np